setup.cfg

```diff
@@ -68,6 +68,7 @@ install_requires =
     solara
     solara-enterprise
     traitlets
+    ipyfilechooser
```

Comment:
This should be pulled in by our `glue-plotly` requirement, now that we've added the `jupyter` extras.

Reply:
Cool, removed!